### PR TITLE
Fix Package.swift

### PR DESCRIPTION
**Pull request by saicu**
Redis/redis-kit can't be build with the newest 5.2 snapshot, redis-kit errors with 
`'redis-kit' /__w/redis/redis/.build/checkouts/redis-kit: error: product dependency 'RediStack' in package 'swift-redi-stack' not found`
see https://github.com/saicu/redis/runs/493164967
I just changed the workflow to use the nightly swift builds.

This should fix it, see https://github.com/saicu/redis/actions/runs/51663070

The tests fail with the actual vapor/swift:5.2-X, not sure why. Changing them to the newest (official) nightly docker images fixes it.

### CODE REVIEW

**Comment by 0xTim** [comment]
@saicu any particular reason you changed the CI file to use nightly etc?

File: .github/workflows/test.yml
Comment (0xTim) [inline .github/workflows/test.yml]: all the commented out code should be deleted but I think we should stick to the vapor images

File: Package.swift
Comment (0xTim) [inline Package.swift:14]: I don't think this is required since it's declared in the packages manifest

**Review by 0xTim — CHANGES_REQUESTED** [review]

**Comment by 0xTim** [comment]
Ah just seen your comment about the nightlies. It's probably a compiler bug that's been fixed - @tanner0101 can we update the 5.2 Vapor images on Docker?

File: Package.swift
Comment (saicu) [reply Package.swift:14]: Without the build fails with actual 5.2 (only testing on Linux, not macOS).
```
'redis-kit' /home/a/Schreibtisch/redis-kit: error: product dependency 'RediStackTestUtils' in package 'swift-redi-stack' not found
'redis-kit' /home/a/Schreibtisch/redis-kit: error: product dependency 'RediStack' in package 'swift-redi-stack' not found
```
Not sure tho if this is the best solution

Comment (saicu) [reply Package.swift:14]: You can see it here: https://github.com/saicu/redis/runs/493164967
Thats the run with the unchanged Package.swift (even if build indirectly through Redis package)

Comment (0xTim) [reply Package.swift:14]: Ok this is definitely weird. I think there's some undefined behaviour in SwiftPM's resolver where the name of the Package doesn't match up with the Git URL. The fact that this builds in Xcode 11.4 Beta 3 suggests that we should get that clarified before merging anything. Do you want to file an issue on bugs.swift.org?

Comment (saicu) [reply Package.swift:14]: @0xTim yep, feels kinda fishy. I made a report here https://bugs.swift.org/browse/SR-12339

Comment (saicu) [reply Package.swift:14]: @0xTim We got a reply there - the changes to the Package.swift are necessary

**Comment by tanner0101** [comment]
@0xTim updated them

**Comment by 0xTim** [comment]
@tanner0101 Thanks, turns out latest 5.2 snapshots have the correct behaviour, it's a problem with the way RediStack is set up. Raised an issue [there](https://gitlab.com/Mordil/swift-redi-stack/-/issues/72)

**Comment by saicu** [comment]
So should RediStack be changed and not this package?

**Comment by 0xTim** [comment]
Yes, I'm under the impression that to make integration easier, RediStack should be changed. But it's up to @Mordil and we can always change this manifest if not

**Comment by Mordil** [comment]
5.2's Package Description is really making it apparent just how much of a weird situation it was between the Package name, Product name, and git URL.

This is really going to force a unification of naming, that will be good in the long run, but very annoying in the short term / painful for long lived projects.

I'll do some renaming of RediStack to improve this... ETA end of day (GMT-8)

**Comment by 0xTim** [comment]
Closing in favour of #31

**Pull request closed without merging**
